chukwuemekachm/M-Tracker-React-Client
# #160919204 Completes the user logout feature

#### What does this PR do?
- This PR completes the user logout feature
#### Description of Task to be completed?
- Add logout action creator in authActions.js
- Add logout case in authReducer.js
- Add tests for user logout in authReducer.test.js
- Update existing snapshots
#### How should this be manually tested?
- Clone the repo
- Startup command line/Terminal
- Navigate to the project's directory
- Run `npm install`
- Run `npm start` or `npm run dev`
- Your default browser will be opened if you used `npm run dev`
- Login or signup(navigate to `/login` or `/signup`)
- Click on the logout button on the navbar
- You will be redirected to the hompage
- Observe the state of the navbar
#### Production URL
- [heroku](https://m-tracker-react.herokuapp.com)
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- [#160919204](https://www.pivotaltracker.com/story/show/160919204)
#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-10-02 at 11 55 44 am" src="https://user-images.githubusercontent.com/33798252/46344919-9d0bd000-c63a-11e8-9356-6c5ee985336e.png">

#### Questions:
- None
